community/events.page.tsx

```diff
@@ -740,7 +740,7 @@ const events = [
     end_date: "September 4, 2024",
   },
   {
-    name: "XRP Community After Hours",
+    name: "XRPL Community After Hours",
```

Comment:
The Linkup entry is called "XRPL Zone Seoul After Hours". Maybe we should match that?

```suggestion
    name: "XRPL Zone Seoul After Hours",
```